src/tests/users.test.ts

```diff
@@ -0,0 +1,203 @@
+  page,
+  loginPage,
+  mainLocation,
+}) => {
+  await editUserPage.authorizationTab.click();
+  await editUserPage.defaultRoleSelect.click();
+  await editUserPage.getUserRole('Manager').click();
+  await editUserPage.defaultLocation.click();
+  const location = await mainLocation.getLocation();
+  await editUserPage.getDefaultLocation(location.name).click();
+  await editUserPage.autoLogin.click();
+  await editUserPage.saveButton.click();
+  await navbar.profileButton.click();
+  await navbar.logoutButton.click();
+  await loginPage.fillLoginForm(formData.username, formData.password);
+  await loginPage.loginButton.click();
+  await expect(page.getByText('My Dashboard')).toBeVisible();
+});
+
+test('Create and impersonate user', async ({ editUserPage, page }) => {
+  await editUserPage.authorizationTab.click();
+  await editUserPage.defaultRoleSelect.click();
+  await editUserPage.getUserRole('Manager').click();
+  await editUserPage.saveButton.click();
+  await editUserPage.impersonateButton.click();
+  const newPage = await page.waitForEvent('popup');
+  const impersonateBanner = new ImpersonateBanner(newPage);
+  await impersonateBanner.isLoaded(formData.username);
+  const newPageNavbar = new Navbar(newPage);
+  await expect(newPageNavbar.configurationButton).toBeHidden();
+  await impersonateBanner.isLoaded(formData.username);
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'Purchasing', exact: true })
+    .click();
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'List Purchase Orders', exact: true })
+    .click();
+  await impersonateBanner.isLoaded(formData.username);
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'Outbound', exact: true })
+    .click();
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'Create Outbound Movement', exact: true })
+    .click();
+  await impersonateBanner.isLoaded(formData.username);
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'Products', exact: true })
+    .click();
+  await newPageNavbar.navbar
+    .getByRole('link', { name: 'List Products', exact: true })
+    .click();
+  await impersonateBanner.isLoaded(formData.username);
+  await newPage.getByRole('button', { name: 'Logout' }).click();
+  await expect(newPage.getByRole('alert')).toBeHidden();
+  await newPage.close();
+});
+
+test('Create user with no access global permissions', async ({
+  navbar,
+  userListPage,
+  editUserPage,
+  loginPage,
+  locationChooser,
+  mainLocation,
+}) => {
+  test.setTimeout(3 * 60 * 1000);
+  await editUserPage.authorizationTab.click();
+  await editUserPage.defaultRoleSelect.click();
+  await editUserPage.getUserRole('No access').click();
+  await editUserPage.saveButton.click();
+  await navbar.profileButton.click();
+  await navbar.logoutButton.click();
+  await loginPage.fillLoginForm(formData.username, formData.password);
+  await loginPage.loginButton.click();
+  await expect(locationChooser.emptyLocationChooser).toBeVisible();
```

Comment:
I belive that this is the point where test _Create user with no access global permissions_ is finished an the following code should be a separate test that checks _Create user with manager global permission_ or _Create user with manager global permission should see depot location in location chooser_